### PR TITLE
Fix #1350: Topic activity back navigation icon

### LINKED ISSUE #1350: Topic Navigation Back Icon
Currently `TopicActivity` contains hamburger icon but this should be replaced with back-arrow navigation icon.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR replaces the `hamburger` icon implementation in `TopicActivity` to `back-arrow` icon.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/topic/TopicViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package org.oppia.app.topic
+
+import androidx.fragment.app.Fragment
```
Comment (veena14cs) [inline app/src/main/java/org/oppia/app/topic/TopicViewModel.kt]: Remove this unused import.

**Review by veena14cs — CHANGES_REQUESTED** [review]
Test case is missing for back navigation icon.

File: app/src/main/java/org/oppia/app/topic/TopicViewModel.kt (hunk 1 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/topic/TopicViewModel.kt]: Done.

**Comment by rt4914** [comment]
> Test case is missing for back navigation icon.

I have added test cases and shifted them to `TopicFragmentTest` and removed `TopicActivityTest` as we do not need that file.

**Review by veena14cs — APPROVED** [review]
LGTM.

**Pull request merged**